db/Models/app.models.js

```diff
@@ -176,6 +176,24 @@ function fetchUsers() {
     });
 }
 
+function removeComment(comment_id) {
+  return db
+    .query(
+      `
+    DELETE FROM comments 
+    WHERE comment_id = $1
+    RETURNING *
+    `,
+      [comment_id]
+    )
+    .then((result) => {
```

Comment:
You could remove the returning here, I think for a delete there is a deleteCount property that comes back from the results object :) but its totes fine 👍